[2025-10-07T02:25:38.376+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: pytorch_circle_classifier.test_model manual__2025-10-07T02:25:19.085669+00:00 [queued]>
[2025-10-07T02:25:38.401+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: pytorch_circle_classifier.test_model manual__2025-10-07T02:25:19.085669+00:00 [queued]>
[2025-10-07T02:25:38.402+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-10-07T02:25:38.402+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-10-07T02:25:38.402+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-10-07T02:25:38.422+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): test_model> on 2025-10-07 02:25:19.085669+00:00
[2025-10-07T02:25:38.452+0000] {standard_task_runner.py:55} INFO - Started process 147 to run task
[2025-10-07T02:25:38.457+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pytorch_circle_classifier', 'test_model', 'manual__2025-10-07T02:25:19.085669+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpqwx3rm9l']
[2025-10-07T02:25:38.468+0000] {standard_task_runner.py:83} INFO - Job 11: Subtask test_model
[2025-10-07T02:25:38.623+0000] {task_command.py:388} INFO - Running <TaskInstance: pytorch_circle_classifier.test_model manual__2025-10-07T02:25:19.085669+00:00 [running]> on host 2db26b28ea99
[2025-10-07T02:25:39.958+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Aditya Ratan
AIRFLOW_CTX_DAG_ID=pytorch_circle_classifier
AIRFLOW_CTX_TASK_ID=test_model
AIRFLOW_CTX_EXECUTION_DATE=2025-10-07T02:25:19.085669+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-10-07T02:25:19.085669+00:00
[2025-10-07T02:25:39.959+0000] {logging_mixin.py:137} INFO - Testing model with new data...
[2025-10-07T02:25:39.979+0000] {logging_mixin.py:137} INFO - Model loaded from ./models/circle_classifier.pth
[2025-10-07T02:25:39.979+0000] {logging_mixin.py:137} INFO - Training accuracy was: 1.0000
[2025-10-07T02:25:39.980+0000] {logging_mixin.py:137} INFO - Test input: x1=0.01, x2=0.2
[2025-10-07T02:25:39.985+0000] {logging_mixin.py:137} INFO - 
--- Prediction Results ---
[2025-10-07T02:25:39.985+0000] {logging_mixin.py:137} INFO - Input: (0.0100, 0.2000)
[2025-10-07T02:25:39.985+0000] {logging_mixin.py:137} INFO - Probability: 0.9959
[2025-10-07T02:25:39.985+0000] {logging_mixin.py:137} INFO - Predicted Label: 1
[2025-10-07T02:25:39.986+0000] {logging_mixin.py:137} INFO - Classification: Inside circle
[2025-10-07T02:25:39.986+0000] {logging_mixin.py:137} INFO - Results saved to ./results/prediction_result.json
[2025-10-07T02:25:39.986+0000] {python.py:177} INFO - Done. Returned value was: {'x1': 0.01, 'x2': 0.2, 'probability': 0.9958621263504028, 'predicted_label': 1, 'label_description': 'Inside circle', 'true_label': 'Inside circle'}
[2025-10-07T02:25:40.010+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=pytorch_circle_classifier, task_id=test_model, execution_date=20251007T022519, start_date=20251007T022538, end_date=20251007T022540
[2025-10-07T02:25:40.052+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-10-07T02:25:40.129+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
